packages/gatsby-plugin-image/src/components/lazy-hydrate.tsx

```diff
@@ -47,6 +59,13 @@ export function lazyHydrate(
   const cacheKey = JSON.stringify(images)
   const hasLoaded = !hydrated.current && hasImageLoaded(cacheKey)
 
+  imgStyle = {
+    objectFit,
+    objectPosition,
+    backgroundColor,
+    ...imgStyle,
+  }
+
   const component = (
     <LayoutWrapper layout={layout} width={width} height={height}>
```

Comment:
This will wrap the image, correct? Can we add options to add className and styles here as well? Or something like `layoutProps`? :)

Reply:
No, it returns a fragment (see layout-wrapper.tsx), so no wrapper DOM element. The wrapper element is created in GatsbyImageHydrator, which does have className and style added.

Reply:
Ok nice!